crates/oxc_formatter/src/write/mod.rs

```diff
@@ -2762,14 +2762,36 @@ impl<'a> FormatWrite<'a> for TSIndexSignature<'a> {
         write!(
             f,
             ["[", self.parameters.first().unwrap(), "]", self.type_annotation, OptionalSemicolon]
-        )?;
+        )
+    }
+}
+
+impl<'a, T, const N: usize> Format<'a> for [&T; N]
```

Comment:
Is this necessary? I don't see where it's used.

Reply:
Removed! This isn't needed now. I tried to use this to solve print `ThisParam` and `FormalParameters` in the same `group`, but it doesn't work.